# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xemacs-beta/max_align.patch
# Copyright (C) 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xemacs-21.5.34/src/lisp.h.vanilla	2017-03-18 20:59:36.400857441 +0000
+++ xemacs-21.5.34/src/lisp.h	2017-03-18 20:59:48.928856927 +0000
@@ -1162,7 +1162,7 @@
   struct { void *p; } p;
   struct { void (*f)(void); } f;
   struct { double d; } d;
-} max_align_t;
+} max_align_t__;
 
 /* ALIGNOF returns the required alignment of a type -- i.e. a value such
    that data of this type must begin at a memory address which is a
